.github/workflows/build.yml

```diff
@@ -130,6 +130,22 @@ jobs:
               comment_id: +COMMENT_ID,
             } );
 
+      - name: Trigger TeamCity build
+        if: ${{ fromJSON(steps.changed.outputs.projects)['plugins/jetpack'] == true }}
+        env:
+          SECRET: ${{ secrets.JETPACK__GITHUB_TEAMCITY_TOKEN }}
```

Comment:
It looks like this might be a personal token @brbrr (watching a build->TC->phab occur now). Not a blocker for merge since the secret can be updated separately.

Reply:
No, this is a secret that is used to verify the request. There are a bunch of other personal tokens that were used, but they'll be updated to shared one by devops.